beacon-chain/blockchain/service.go

```diff
@@ -8,6 +8,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+
```

Comment:
we should remove the spacing here? stupid IDE... 😞

Reply:
yes